docs/getting-started/getting-started-darwin.md

```diff
@@ -164,6 +164,9 @@ git clone https://github.com/facebook/rocksdb.git
 cd rocksdb
 git checkout v6.15.5
 
+# Apply the RocksDB patch in this repo to support newer versions of gcc
```

Comment:
Question: Should we have the patch in stackrox/stackrox as well?
We're setting a precedent for having the patch in this documentation repo.
I'm fine with this, but it also means you definitely need to have a look in this repo as well even if you only want to start working on stackrox/stackrox.